zone/bot.h

```diff
@@ -87,7 +93,125 @@ enum BotCastingChanceConditional : uint8
 	cntHSND = 16
 };
 
+namespace BotSettingCategories { // Update GetBotSpellCategoryName as needed
+	constexpr uint8 BaseSetting							= 0;
+	constexpr uint8 SpellHold							= 1;
+	constexpr uint8 SpellDelay							= 2;
+	constexpr uint8 SpellMinThreshold					= 3;
+	constexpr uint8 SpellMaxThreshold					= 4;
+	constexpr uint8 SpellTypeAggroCheck					= 5;
+	constexpr uint8 SpellTypeMinManaPct					= 6;
+	constexpr uint8 SpellTypeMaxManaPct					= 7;
+	constexpr uint8 SpellTypeMinHPPct					= 8;
+	constexpr uint8 SpellTypeMaxHPPct					= 9;	
+	constexpr uint8 SpellTypeIdlePriority				= 10;
+	constexpr uint8 SpellTypeEngagedPriority			= 11;
+	constexpr uint8 SpellTypePursuePriority				= 12;
+	constexpr uint8 SpellTypeAEOrGroupTargetCount		= 13;
+	constexpr uint8 SpellTypeRecastDelay				= 14;
+
+	constexpr uint16 START								= BotSettingCategories::BaseSetting;
+	constexpr uint16 START_NO_BASE						= BotSettingCategories::SpellHold;
+	constexpr uint16 START_CLIENT						= BotSettingCategories::SpellHold;
+	constexpr uint16 END_CLIENT							= BotSettingCategories::SpellMaxThreshold;
+	constexpr uint16 END								= BotSettingCategories::SpellTypeAEOrGroupTargetCount;
+	constexpr uint16 END_FULL							= BotSettingCategories::SpellTypeRecastDelay;
```

Comment:
This is not quite aligned

Reply:
These for me show up properly in VS, even on another PC to compare but I do see them misaligned on Github. Do you happen to have an easy solution without pushing commits to see how it reflects?